.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Can you add IDE specific files to [your global `.gitignore`](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)?